From 62b9a623b8860b85817251efb3ecca528e990b39 Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Wed, 27 Apr 2016 16:29:31 +0200
Subject: [PATCH 1/2] wsgiServer: Reduce the socket poll time

The wsgi server seems to be waiting for 1 second between polls.
---
 wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py | 2 +-
 wsgidav/server/cherrypy/wsgiserver/wsgiserver3.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py b/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
index cd1ac2f..cdf24ab 100644
--- a/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
+++ b/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
@@ -1844,7 +1844,7 @@ class HTTPServer(object):
             raise socket.error(msg)
 
         # Timeout so KeyboardInterrupt can be caught on Win32
-        self.socket.settimeout(1)
+        self.socket.settimeout(0.1)
         self.socket.listen(self.request_queue_size)
 
         # Create worker threads
diff --git a/wsgidav/server/cherrypy/wsgiserver/wsgiserver3.py b/wsgidav/server/cherrypy/wsgiserver/wsgiserver3.py
index a47f8d5..3e2064a 100644
--- a/wsgidav/server/cherrypy/wsgiserver/wsgiserver3.py
+++ b/wsgidav/server/cherrypy/wsgiserver/wsgiserver3.py
@@ -1550,7 +1550,7 @@ class HTTPServer(object):
             raise socket.error(msg)
 
         # Timeout so KeyboardInterrupt can be caught on Win32
-        self.socket.settimeout(1)
+        self.socket.settimeout(0.1)
         self.socket.listen(self.request_queue_size)
 
         # Create worker threads
-- 
2.7.4

